Bonsai.System/IO/GetEnvironmentVariable.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Bonsai;
```

Comment:
This using should not be needed as the operator is declared inside the `Bonsai` namespace.